packages/central-server/src/dhis/DhisChangeValidator.js

```diff
@@ -41,21 +45,71 @@ export class DhisChangeValidator extends ChangeValidator {
               ? `AND ${this.models.surveyResponse.getExcludeEventsQueryClause()}`
               : ''
           }
+          AND survey_response.outdated = ?
           AND survey_response.id IN (${batchOfSurveyResponseIds.map(() => '?').join(',')});
         `,
-        [...nonPublicDemoLandUsers, ...batchOfSurveyResponseIds],
+        [...nonPublicDemoLandUsers, includeOutdated, ...batchOfSurveyResponseIds],
       );
       validSurveyResponseIds.push(...batchOfSurveyResponses.map(r => r.id));
     }
     return validSurveyResponseIds;
   };
 
+  getOutdatedAnswers = async surveyResponseIds => {
+    // get the answers associated with the survey responses
+    const associatedAnswers = await this.models.answer.find({
+      survey_response_id: surveyResponseIds,
+    });
+
+    // filter out answers for questions that do not sync to dhis
+    const dhisLinkedAnswers = await this.filterDhisLinkedAnswers(associatedAnswers);
+
+    // create delete changes for the answers
+    const deleteAnswers = await Promise.all(
+      dhisLinkedAnswers.map(async a => ({
+        record_type: 'answer',
+        record_id: a.id,
+        type: 'delete',
+        new_record: null,
+        old_record: await a.getData(),
+      })),
+    );
+
+    return deleteAnswers;
```

Comment:
We might want to test to check that this code doesn't lead to us attempting to delete the same answer twice if it the survey response changes. Admittedly we're moving away from changing survey responses in favour of outdating but it does still happen.

Looking at this code, it does seem like we'd attempt to delete it twice, and if so that might be a problem. A fix would be to not push a delete to the dhis sync queue if there's already one there?